dlp/inspect.js

```diff
@@ -15,167 +15,140 @@
-  });
-}
+  // The string to inspect
+  // const string = 'My name is Gary and my email is gary@example.com';
 
-// Helper function to get results of a long-running (polling-required) job
-function getJobResults (authToken, jobName) {
-  // Construct REST request to get results of finished inspect job
-  const options = {
-    url: `${API_URL}/inspect/results/${jobName}/findings`,
-    headers: {
-      'Authorization': `Bearer ${authToken}`,
-      'Content-Type': 'application/json'
-    },
-    json: true
-  };
+  // The minimum likelihood required before returning a match
+  // const minLikelihood = LIKELIHOOD_UNSPECIFIED;
 
-  // Run job-results-fetching REST request
-  return requestPromise.get(options);
-}
+  // The maximum number of findings to report (0 = server maximum)
+  // const maxFindings = 0;
 
-function inspectString (authToken, string, inspectConfig) {
-  // [START inspect_string]
-  // Your gcloud auth token
-  // const authToken = 'YOUR_AUTH_TOKEN';
+  // The infoTypes of information to match
+  // const infoTypes = ['US_MALE_NAME', 'US_FEMALE_NAME'];
 
-  // The string to inspect
-  // const string = 'My name is Gary and my email is gary@example.com';
+  // Whether to include the matching string
+  // const includeQuote = true;
 
   // Construct items to inspect
   const items = [{ type: 'text/plain', value: string }];
 
-  // Construct REST request body
-  const requestBody = {
+  // Construct request
+  const request = {
     inspectConfig: {
-      infoTypes: inspectConfig.infoTypes,
-      minLikelihood: inspectConfig.minLikelihood,
-      maxFindings: inspectConfig.maxFindings,
-      includeQuote: inspectConfig.includeQuote
+      infoTypes: infoTypes,
+      minLikelihood: minLikelihood,
+      maxFindings: maxFindings,
+      includeQuote: includeQuote
     },
     items: items
   };
 
-  // Construct REST request
-  const options = {
-    url: `${API_URL}/content:inspect`,
-    headers: {
-      'Authorization': `Bearer ${authToken}`,
-      'Content-Type': 'application/json'
-    },
-    json: requestBody
-  };
-
-  // Run REST request
-  requestPromise.post(options)
-    .then((body) => {
-      const results = body.results[0].findings;
-      console.log(JSON.stringify(results, null, 2));
+  // Run request
+  dlp.inspectContent(request)
+    .then((response) => {
+      const findings = response[0].results[0].findings;
+      if (findings.length > 0) {
+        console.log(`Findings:`);
+        findings.forEach((finding) => {
+          if (includeQuote) {
+            console.log(`\tQuote: ${finding.quote}`);
+          }
+          console.log(`\tInfo type: ${finding.infoType.name}`);
+          console.log(`\tLikelihood: ${finding.likelihood}`);
+        });
+      } else {
+        console.log(`No findings.`);
+      }
     })
     .catch((err) => {
-      console.log('Error in inspectString:', err);
+      console.log('Error in inspectString:', err.message || err);
```

Comment:
nit: Why not: `Error in inspectString: ${err.message || err}` to be consistent with other logging style.